src/index.css

```diff
@@ -14,3 +14,7 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
+
+summary::marker {
+  content: '';
+}
```

Comment:
If we end up removing the details-approach in the subcategory table, we can remove this custom styling as well